src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,32 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example of Import order:
+ * </p>
+ * <pre>
+ * import java.io.IOException; //OK
+ * import java.net.URL; //OK
+ * import java.security.KeyManagementException; //OK
+ *
+ * import javax.net.ssl.SSLContext; //OK
```

Comment:
Make sure to leave a space after comment delimiter:
like this: `// OK`, not `//OK`.

Reply:
done.